.github/CODEOWNERS

```diff
@@ -570,8 +559,10 @@
 
 
 # User Data Takeout.
-/core/domain/takeout_*.py @BenHenning @vojtechjelinek
-/core/domain/wipeout_*.py @BenHenning @vojtechjelinek
+# TODO(#11477): Restore codeowners to @BenHenning after 2021-01-04.
```

Comment:
@BenHenning Please add me to both of these too.

Reply:
Done.